### PR TITLE
[MINOR] EDA-1297 board to string

### PR DESCRIPTION
**Pull request by ibrizuela-eb**
I need help to refactor the process to parse a board into a string. It's working but it needs a CODE-REVIEW

### CODE REVIEW

**Review by sarias-eb — APPROVED** [review]
LGTM

File: game/board.py [hunk 1]
```diff
@@ -23,7 +23,51 @@ class Board:
...
+
+    def setup_new_board(self):
+        self.place_pawn(self.pawns['red'], 0, 4)
+        self.place_pawn(self.pawns['blue'], 8, 4)
+
+    def get_board_str(self) -> str:
+        board_str = ''
+        for spot_str, wall_spot in zip(self.spots_to_str(), self.walls_to_str()):
+            board_str += spot_str + wall_spot
+        board_str += self.spots_to_str()[-1]
```
Comment (nicolasnunez-eb) [inline game/board.py:39]: Another way to do this is putting a string at the end of `walls_to_str` like `''`

**Review by nicolasnunez-eb — APPROVED** [review]
NIT comment but lgtm

File: game/board.py (hunk 1 above)
Comment (ibrizuela-eb) [reply game/board.py:39]: I thought about it but in fact that `''` I'll be adding an element that is not a spot for wall.
I mean these are all posible spot for walls.
```
' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
            '                 ',
            ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
            '                 ',
            ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
            '                 ',
            ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
            '                 ',
            ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
            '                 ',
            ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
            '                 ',
            ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
            '                 ',
            ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
            '                 ',
            ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ',
`
Do you understand?

**Pull request merged**